src/routes/docs/+page.svelte

```diff
@@ -1,5 +1,7 @@
 <script lang="ts">
   import DocsMain from './components/DocsMain.svelte';
+  import Explore from './components/Explore.svelte';
 </script>
 
 <DocsMain />
+<Explore classes="mt-4" />
```

Comment:
```suggestion
<Explore />
```
This is currently not doing anything and I don't think it's necessary. if we want to add spacing, i would do it inside of the component or at the top level using something like `gap`